.zappr.yaml

```diff
@@ -0,0 +1,6 @@
+approvals:
+  minimum: 2
```

Comment:
would add this:

```
  from:
    collaborators: true
```

or is this the default?

(https://zappr.readthedocs.io/en/latest/setup/)